src/test/regress/Makefile

```diff
@@ -48,6 +52,25 @@ check: check-full
 # check-full triggers all tests that ought to be run routinely
 check-full: check-multi check-multi-mx check-multi-task-tracker-extra check-worker check-follower-cluster check-failure
 
+
+ISOLATION_DEPDIR=.deps/isolation
+ISOLATION_BUILDDIR=build/specs
+
+# this can be used to print a value of variable
+# ex: make print-generated_mx_files
+print-%  : ; @echo $* = $($*)
+
+$(ISOLATION_DEPDIR) $(ISOLATION_BUILDDIR):
+	mkdir -p $(citus_abs_srcdir)/$@
```

Comment:
Directories as targets mess make up. I suspect because modified time of directories can change even though we only care about the fact that it exists. So simply move this to the generated_isolation_fiels rule: https://github.com/citusdata/citus/pull/2977/files